chsdi/views/shortener.py

```diff
@@ -31,7 +31,8 @@ def _add_item(table, url):
                     'url_short': url_short,
                     'url': url,
                     'timestamp': time.strftime('%Y-%m-%d %X', now),
-                    'epoch': time.strftime('%s', now)
+                    'epoch': time.strftime('%s', now),
+                    'day': time.strftime('%Y-%m-%d', now),
```

Comment:
Would not be `'date'` as field name more appropriate here ?

Reply:
New key `staging`